services/workflows-service/src/workflow-defintion/workflow-definition.controller.internal.ts

```diff
@@ -0,0 +1,19 @@
+import * as common from '@nestjs/common';
+import * as swagger from '@nestjs/swagger';
+import { ApiExcludeController } from '@nestjs/swagger';
+import * as errors from '../errors';
+import { WorkflowDefinitionService } from '@/workflow-defintion/workflow-definition.service';
+import { CreateDemoWorkflowDefinitionDto } from '@/workflow-defintion/dtos/create-demo-workflow-definition-dto';
+
+@ApiExcludeController()
+@common.Controller('internal/workflow-definition')
+export class WorkflowControllerInternal {
+  constructor(protected readonly service: WorkflowDefinitionService) {}
+
+  @common.Post('/create-demo')
+  @swagger.ApiOkResponse()
+  @swagger.ApiForbiddenResponse({ type: errors.ForbiddenException })
+  async createDemoWorkflowDefinition(@common.Body() data: CreateDemoWorkflowDefinitionDto) {
```

Comment:
If you want to access this endpoint as an admin, I think you have to add `@common.UseGuards(AdminAuthGuard)` decorator here.